sample-workflows/test-build.yml

```
@@ -0,0 +1,60 @@
+name: Test Build Workflow
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  push:
+    branches:
+      - '*'
+  workflow_dispatch:
+
+jobs:
+  build-translation:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        version: [ 3.13 ]
+        format: [ html, latex ]
+    steps:
+      - uses: actions/setup-python@master
+        with:
+          python-version: 3.12  # pinned for Sphinx 3.4.3 to build 3.10
+      - uses: actions/checkout@master
+        with:
+          repository: python/cpython
+          ref: ${{ matrix.version }}
+      - run: make venv
+        working-directory: ./Doc
+      - uses: actions/checkout@master
+        with:
+          ref: ${{ matrix.version }}
+          path: Doc/locales/XX/LC_MESSAGES
+      - run: git pull
+        working-directory: ./Doc/locales/XX/LC_MESSAGES
+      - uses: sphinx-doc/github-problem-matcher@v1.1
+      - run: make -e SPHINXOPTS="--color -D language='XX' -W --keep-going" ${{ matrix.format }}
+        working-directory: ./Doc
+      - uses: actions/upload-artifact@master
+        if: success() || failure()
+        with:
+          name: build-${{ matrix.version }}-${{ matrix.format }}
+          path: Doc/build/${{ matrix.format }}
+
+    output-pdf:
+      runs-on: ubuntu-latest
+      strategy:
+        matrix:
+          version: [ 3.13 ]
```

python-newest now reflects the 3.14 release.

```suggestion
          version: [ '3.14' ]
```